composer.json

```diff
@@ -14,11 +14,11 @@
         "illuminate/support": "^5.7",
         "jean85/pretty-package-versions": "^1.2",
         "latte/latte": "^2.4",
-        "nette/di": "^2.4",
+        "nette/di": "^2.4|^3.0@rc",
```

Comment:
This cannot go to tagged version and I'd definitely forget to check it before release. Only stable tags.

Reply:
Ok, we are going to wait.

Reply:
Why not keep 2.4?

Reply:
I have done this, because of trying ECS in my project with Nette v3. With 2.4 I am getting conflict.

Reply:
I see. This is not how you solve it.

Edit your `composer.json`:

```json
{
    "require": {
        "nette/di": "3.0 as 2.4"
    },
    "prefer-stable": true,
    "minimal-stability": "dev"
}
```